src/ast/mod.rs

```diff
@@ -1276,9 +1280,18 @@ impl fmt::Display for Statement {
                 //   `CREATE TABLE t (a INT) AS SELECT a from t2`
                 write!(
                     f,
-                    "CREATE {or_replace}{external}{temporary}TABLE {if_not_exists}{name}",
+                    "CREATE {or_replace}{external}{global}{temporary}TABLE {if_not_exists}{name}",
                     or_replace = if *or_replace { "OR REPLACE " } else { "" },
                     external = if *external { "EXTERNAL " } else { "" },
+                    global = if global.is_some() {
+                        if global.unwrap() == true {
+                            "GLOBAL "
+                        } else {
+                            "LOCAL "
+                        }
+                    } else {
+                        ""
+                    },
```

Comment:
I think more idomatic rust would be the following (it avoids `unwrap` when you know it is already `Some`)

```suggestion
                    global = global.map(|global| {
                        if global.unwrap() == true {
                            "GLOBAL "
                        } else {
                            "LOCAL "
                        }
                    }).unwrap_or(""),
```